package.json

```diff
@@ -1332,10 +1332,14 @@
           "default": "glob",
           "enum": [
             "glob",
+            "globActive",
+            "globRoot",
             "command"
           ],
           "markdownEnumDescriptions": [
             "Clean all the files located in `#latex-workshop.latex.outDir#` and matching the glob patterns listed in `#latex-workshop.latex.clean.fileTypes#`.",
+            "Clean all the files located in `#latex-workshop.latex.outDir#` and matching the glob patterns listed in `#latex-workshop.latex.clean.fileTypes#`. The cleaned file will have the same as the current active LaTeX file. If the active one is not LaTeX, this config falls back to `globRoot`.",
+            "Clean all the files located in `#latex-workshop.latex.outDir#` and matching the glob patterns listed in `#latex-workshop.latex.clean.fileTypes#`. The cleaned file will have the same as the jobname or root file. If the root file is not defined, the cleaner will do nothing.",
```

Comment:
I guess the word _name_ or _base name_ is missing in 

- The cleaned file will have the same as the current active LaTeX file
- The cleaned file will have the same as the jobname or root file